src/Diagnostics.RuntimeHost/Services/DevOpsClient/DevOpsClient.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Microsoft.VisualStudio.Services.WebApi;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Diagnostics.RuntimeHost.Services.DevOpsClient
+{
+    public interface IDevOpsClient : IDisposable, IRepoClient
+    {
+        string AccessToken { get; }
+        string Organization { get; }
+        string RepositoryID { get; }
+        string Project { get; }
+    }
+
+    public class DevOpsResponse
+    {
+        public object result;
+        public HttpStatusCode responseCode;
+    }
+
+    class DevOpsClient : IDevOpsClient
+    {
+        private static string _accessToken;
+        private static string _organization;
+        private static string _repoID;
+        private static string _project;
+        private static IConfiguration _config;
+        private static VssCredentials credentials;
+        private static VssConnection connection;
+        private GitHttpClient gitClient;
+
+        public DevOpsClient(IConfiguration config)
+        {
+            _config = config;
```

Comment:
Is `_config` used anywhere else? I see that you are passing constructor parameter `config` to `LoadConfigurations` method

Reply:
It does not. I meant to remove it.